sdk/python/tests/infra/offline_stores/test_helpers.py

```diff
@@ -0,0 +1,24 @@
+import pytest
```

Comment:
Unsure why I get an import statement when adding those tests ... 🤔 
```
sdk/python/feast/client.py:26: in <module>
    from feast.data_source import BigQuerySource, FileSource
sdk/python/feast/data_source.py:20: in <module>
    from feast.infra.offline_stores.bigquery import BigQueryOfflineStore
sdk/python/feast/infra/offline_stores/bigquery.py:11: in <module>
    from feast.data_source import BigQuerySource, DataSource
E   ImportError: cannot import name 'BigQuerySource' from 'feast.data_source' (/Users/mattdelacour/src/github.com/Shopify/feast/sdk/python/feast/data_source.py)
```

Reply:
Cycling dependency?

Reply:
Ho yeah it sounds like it ... 😞 

Will have a look tomorrow

Reply:
Done

Reply:
So my understanding is 
DataSource (from `data_source.py`) is meant to be a low level interface. Then "stores" lile BigQueryOfflineStore can use data sources.

But when I look at the references, BigQuerySource is used everywhere. So it was a bit tricky to bypass the cyclic dependency (I basically make the import statement at the property level which is not great but could not think about another option)

Happy to get you opinion on this one 🙂

Reply:
> But when I look at the references, BigQuerySource is used everywhere. So it was a bit tricky to bypass the cyclic dependency (I basically make the import statement at the property level which is not great but could not think about another option)

Are there any areas where you see us using BigQuerySource where you think it should not occur?

Cyclic dependencies generally (although not always) point to a problem with code.